pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -234,6 +232,14 @@ message ServerConfiguration {
   //
   // More details: https://github.com/kubernetes/enhancements/issues/753
   bool stop_gracefully = 12;
+
+  oneof transport_security {
+      // TLS configuration. TLS is not enabled when left unset.
+      buildbarn.configuration.tls.ServerConfiguration tls = 3;
+      // If set, enable Application Layer Transport Security (ALTS).
```

Comment:
Given that ALTS isn't widely known outside of Google/GCE, could you please add a link to https://docs.cloud.google.com/docs/security/encryption-in-transit/application-layer-transport-security as well?